### PR TITLE
Standardize end-of-task message

### PR DESCRIPTION
**Pull request by zwatson2001**
The goal of this PR is to display the same message at the end of each task (inside a prompt box with audio) regardless of whether it ended early due to mistakes. At the moment the screen displayed after making it all the way through a task has an exit button, but the other screen doesn't.

### CODE REVIEW

**Comment by zwatson2001** [comment]
I updated the timelines of each task so that they skip all trials once max incorrect is reached (so that the last trial with the end message + exit button is always shown at the end).

**Comment by asengupta3** [comment]
@zwatson2001 is there a reason that you are skipping trials with conditionals as opposed to finishing the whole experiment?

File: task-launcher/src/tasks/shared/trials/finishExperiment.js [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+    ); 
+
+    window.addEventListener('click', (event) => {
+        const buttonId = event.target.id;
+        if (buttonId === 'exit-button') {
+            document.body.innerHTML = '';
+        }
+    }); 
+
+    document.addEventListener('keydown', () => {
```
Comment (asengupta3) [inline task-launcher/src/tasks/shared/trials/finishExperiment.js:26]: This might add a permanent keydown event to destroy the entire html every time a key is placed. Why is this needed?

File: task-launcher/src/tasks/shared/trials/finishExperiment.js [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
+            </div>
+            <footer>${t.generalFooter}</footer>
+            <button id="exit-button" class="primary" style=margin-top:5%>Exit</button>
+        </div>`,
+        PageAudioHandler.playAudio(mediaAssets.audio.taskFinished)
+    ); 
+
+    window.addEventListener('click', (event) => {
+        const buttonId = event.target.id;
+        if (buttonId === 'exit-button') {
```
Comment (asengupta3) [inline task-launcher/src/tasks/shared/trials/finishExperiment.js:21]: I forgot to mention that we should probably also remove the listener. So my suggestion would be to move this listener to before `jsPsych.endExperiment` is declared and remove the listener once the button is placed.
```
const removeDOMElements = (event) => {
        const buttonId = event.target.id;
        if (buttonId === 'exit-button') {
           document.body.innerHTML = '';
            // clean up so that it does not persist
           window.removeEventListener('click', removeDOMElements);
        }
}
window.addEventListener('click', removeDOMElements);
```

File: task-launcher/src/tasks/shared/trials/finishExperiment.js [hunk 3]
```diff
@@ -1,12 +1,29 @@
...
 
 
 export function finishExperiment() {
     const t = taskStore().translations;
     jsPsych.endExperiment(
-      `<div id="prompt-container-text">
-                <p id="prompt">${t?.taskCompletion || 'Thank you for completing the task! You can now close the window.'}</p>
-            </div>`,
-    ); // ToDo: style text and add audio message?
+        `<div class='instructions-container'>
```
Comment (asengupta3) [inline task-launcher/src/tasks/shared/trials/finishExperiment.js:9]: Can you replace this with `lev-stimulus-container`? We want to get rid of all `task.scss` declarations.

File: task-launcher/src/tasks/shared/trials/taskFinished.js [hunk 4]
```diff
@@ -13,13 +14,17 @@ export const taskFinished = {
     const t = taskStore().translations;
     return `
         <div class='instructions-container'>
+          <div class='lev-row-container instruction'>
             <h1 class='instructions-title'>${t.taskFinished}</h1>
```
Comment (asengupta3) [inline task-launcher/src/tasks/shared/trials/taskFinished.js:18]: I think the class declaration can be removed.

File: task-launcher/src/tasks/shared/trials/finishExperiment.js (hunk 3 above)
Comment (zwatson2001) [reply task-launcher/src/tasks/shared/trials/finishExperiment.js:9]: Sure! I forgot to check for that on this file, thanks for the reminder

**Comment by kachergis** [comment]
Hi both, thanks for all your work on this! What's the status here?

**Comment by asengupta3** [comment]
@kachergis it is very close. There is just one more suggestion, I have. After that this will be ready to merge.

**Review by asengupta3 — APPROVED** [review]
LGTM! Thanks Zach for working on this and simplifying the logic. Please feel free to merge whenever you think you are done testing.
